### DISCUSSION TITLE
Will the next version still support Vuepress?

### DISCUSSION THREAD

**Discussion by CaiWenlie** [tgreyuk/typedoc-plugin-markdown#460] [Help] [answered]
or switch to vitepress?

**Comment by tgreyuk** [comment, marked as answer]
Hello. Not sure yet - the plan is for the Vitepress implementation to also support Vuepress (if required) but not through a VuePress plugin. See https://github.com/tgreyuk/typedoc-plugin-markdown/tree/next-dev/packages/typedoc-vitepress-theme#what-about-vuepress. I'll add a working example somewhere.
